docs/operations/setting_up_the_swarm.rst

```diff
@@ -170,3 +176,83 @@ files to the server.
      --entrypoints='Name:https Address::443 TLS:/etc/ssl/certs/openstax.crt,/etc/ssl/private/openstax.pem' \
      --logLevel=INFO \
      --accessLog
+
+Add cronjob to run docker prune
+===============================
+
+Docker swarm does not come with any kind of "garbage collection" for dangling 
+volumes or unused containers that are created when doing updates or after 
+restarts. This has caused issues where the host nodes run out of hard drive storage. To 
+prevent this we have created an :term:`Ansible` playbook to configure a cronjob on the server.
+
+Local or from bastion2?
+----------------------
+
+There are two places you can run this playbook.
+
+1. From ``localhost`` if you have ``bastion2`` setup as a :term:`JumpHost` and proper key 
+   to cops servers in the correct directory.
+2. From ``bastion2`` directly.
+
+See this `guide <https://www.tecmint.com/access-linux-server-using-a-jump-host>`_ 
+to learn how to configure a :term:`JumpHost`.
+
+If you are running from ``bastion2`` you will need to clone down the 
+output-producer-service repository into your home directory and execute the commands.
+
+Running the playbook
+--------------------
+
+* Ensure you are in the root directory of  the project and change directory into 
+  the ``./ansible`` directory.
+
+.. code-block:: bash
+
+   cd ./ansible
+
+* Create a virtualenv for installing `Ansible <https://docs.ansible.com/ansible/latest/index.html>`_ and dependencies
+
+.. code-block:: bash
+
+   python -m .venv venv
+
+* Activate the virtualenv
+
+.. code-block:: bash
+
+   source ./.venv/bin/activate
+
+* Install requirements.txt
```

Comment:
I believe your current requirements.txt is empty / missing the ansible dependency in the file